en/lessons/specifics/plug.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 2.1.1
+version: 2.1.2
```

Comment:
@timrogers since this is a more significant change, let's bump this to `2.2.0`. We've never nailed down the versioning (sorry!) but the gist is:

MAJOR: A whole new lesson or rewrite
MINOR: Changes to content
PATCH: Small typos, grammar

The versions are used to generate reports for the translators. The levels indicate amount of changes. @gemantzu put the report together and manages the Greek translations, check out their report here: https://elixirschool.com/gr/report/